server/app/models/fusor/deployment.rb

```diff
@@ -12,7 +12,6 @@
 
 module Fusor
   class Deployment < ActiveRecord::Base
-    serialize :rhev_params, Hash
     serialize :cfme_params, Hash
     serialize :openstack_params, Hash
```

Comment:
Since we've decided that we won't be using serialized hashes, we can probably go ahead and remove the cfme and openstack hashes.